doc/src/manual/noteworthy-differences.md

```diff
@@ -80,7 +80,8 @@ may trip up Julia users accustomed to MATLAB:
     `x[x.>3]` and `x = x[x.>3]`. Using [`filter!`](@ref) reduces the use of temporary arrays.
   * The analogue of extracting (or "dereferencing") all elements of a cell array, e.g. in `vertcat(A{:})`
     in MATLAB, is written using the splat operator in Julia, e.g. as `vcat(A...)`.
-
+  * In Julia, the `adjoint` function performs conjugate transposition; in MATLAB, `adjoint` provides the "adjugate" or       
```

Comment:
This has trailing whitespace that was caught by the whitespace check build, but the failure there was apparently ignored. This has broken the whitespace build for all PRs against master.

Reply:
Will be fixed by #33439